sdk/eventhub/eventhubs-checkpointstore-table/test/tables-checkpointstore.spec.ts

```diff
@@ -0,0 +1,518 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import chai from "chai";
+import * as dotenv from "dotenv";
```

Comment:
dotenv is being loaded and configured in your testUtils file, so shouldn't be needed here.

Reply:
addressed